hera_mc/sys_handling.py

```diff
@@ -45,6 +45,39 @@ def cofa(self):
         cofa = self.geo.cofa()
         return cofa
 
+    def get_dubitable_list(self, date='now', return_full=False):
+        """
+        Returns start_time and a list with the dubitable antennas.
+        If date is supplied, returns for that date, otherwise now.
+
+        Parameters:
+        ------------
+        date:  something understandable by cm_utils.get_astropytime
+        return_full:  boolean to return the full object as opposed to the csv string
```

Comment:
The return value needs to be documented more clearly, breaking out the relevant cases (I count three: None, str, tuple) and the semantics of each (including the individual elements when a tuple is returned).

It also seems like it might just be simpler to obey the "return_full" semantics always. The caller can ignore the boundary times and re-join the array if strings if that's what they want to do.

Reply:
This was a specific request to provide back exactly what rtp is looking for.  I wanted the ability to provide more for flexibility and didn't want another function so added return_full.  I've rewritten to try and be clearer.

Reply:
OK.